chains/bitlayer/metadata.yaml

```diff
@@ -0,0 +1,26 @@
+blockExplorers:
+  - apiUrl: https://api.btrscan.com/scan/api
+    family: other
+    name: Bitlayer Explorer
+    url: https://www.btrscan.com
+blocks:
+  confirmations: 1
+  estimateBlockTime: 3
+  reorgPeriod: 0
+chainId: 200901
+deployer:
+  name: Abacus Works
+  url: https://www.hyperlane.xyz
+displayName: Bitlayer
+domainId: 200901
+gasCurrencyCoinGeckoId: bitcoin
+name: bitlayer
+nativeToken:
+  decimals: 18
+  name: Bitcoin
+  symbol: BTC
```

Comment:
This may be the first BTC in the registry, cool to see it

Reply:
second I think - merlin might be the first https://github.com/hyperlane-xyz/hyperlane-registry/blob/main/chains/merlin/metadata.yaml#L16